src/tests/ftest/deployment/target_failure.yaml

```diff
@@ -0,0 +1,72 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
```

Comment:
Any reason to not use one more server or client node? Since HW Medium has 4 nodes available

Reply:
Increased to 4.

Reply:
Thanks! In general, my reasoning for this is about fully utilizing resources available and also not wasting resources. So for most tests (if feasible) I think it makes sense to either:
1. Use more servers/clients so the entire cluster is in use. This sometimes can expand coverage without actually using more resources, since the entire cluster is in use anyway.
2. Use fewer servers/clients so the test can fit on a smaller cluster. This is nice when adding more nodes doesn't improve test coverage - so it makes sense to use as few resources as possible.

Reply:
Right. I'll consider more about utilizing the right number of nodes.